changelogs/unreleased/use-streams-in-binary-console.md

```diff
@@ -0,0 +1,3 @@
+## feature/console
```

Comment:
Nit: Should be `feature/core`. Also, please, for the record, please file a ticket describing the problem you fix.